packages/pci-sdk-iframe/src/pages/app/themes/light.ts

```diff
@@ -83,4 +83,20 @@ export default {
 		boxSizing: 'border-box',
 		margin: '0',
 	},
+	// Notification
+	notification: {
+		fontFamily: 'sans-serif',
+		padding: '2vw 4vw',
+		lineHeight: '24px',
+		position: 'absolute',
+		left: '50%',
+		transform: 'translateX(-50%)',
+		backgroundColor: '#CFD8DC',
+		color: '#000',
+		borderRadius: '6px',
+		zIndex: 3,
+		width: '70%',
+		textAlign: 'center',
+		top: '15px',
```

Comment:
The styles for the light theme would be the following:
```suggestion
	position: 'absolute',
	zIndex: 3,
	top: 4.706vw;
	left: '50%',
	width: '70%',
	padding: '2vw 4vw',
	transform: 'translateX(-50%)',	
	borderRadius: '8px',
	backgroundColor: '#CBCBCB',
	color: '#CFD8DC',
	fontFamily: 'sans-serif',
	font-size: 4.118vw;
  line-height: 1.5;
  textAlign: 'center',
```